src/addons/manager/addonmanager.cpp

```diff
@@ -174,11 +172,9 @@ bool AddonManager::loadManifest(const QString& manifestFileName) {
       }
       if (!enabled) {
```

Comment:
```suggestion
      // Note that `removeRow` and `insertRow` are not called between the `begin`/`end` calls below.
      // Those are not necessary, because the modelling of the addons list in this class is different from what
      // we need to convey to QML i.e. in here the addons list contails all addons while QML needs
      // enabled addons only. See the `AddonManager::rowCount` and `AddonManager::data` for more clarity.
      //
      // The `begin`/`end` calls are still necessary though to trigger specific signals for QML to
      // know that the list has changed and update the UI accordingly.
      if (!enabled) {
```